From 1469be096eb74b1e1e4ca7fc134d1b9b5483453d Mon Sep 17 00:00:00 2001
From: Andreas Schmidt <andreas.schmidt@wago.com>
Date: Thu, 19 Jul 2018 13:58:16 +0200
Subject: [PATCH] dts: fix USB not initiate bug on 8207 devices

The ti,am3517-evm compatible string got lost by rename of dts-tree
in commit a88f655c29213a9c2885d9697f268b4c31335ea0.The "ti,am3517-evm"
compatible string enables the calling of am3517_evm_legacy_init.
The function am3517_evm_legacy resets emac and initiate musb. This
initialization of musb is needed for 8207 with USB 3G Modem. This patch
add the missing compatible string back to dts for all am35xx based PFC's.

Signed-off-by: Peter Galka <petet.galka@wago.com>
Signed-off-by: Andreas Schmidt <andreas.schmidt@wago.com>
---
 arch/arm/boot/dts/am3505-pfc-750_8202.dts | 2 +-
 arch/arm/boot/dts/am3505-pfc-750_8203.dts | 2 +-
 arch/arm/boot/dts/am3505-pfc-750_8204.dts | 2 +-
 arch/arm/boot/dts/am3505-pfc-750_8206.dts | 2 +-
 arch/arm/boot/dts/am3505-pfc-750_8207.dts | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/am3505-pfc-750_8202.dts b/arch/arm/boot/dts/am3505-pfc-750_8202.dts
index 3ed8803..58befe8 100644
--- a/arch/arm/boot/dts/am3505-pfc-750_8202.dts
+++ b/arch/arm/boot/dts/am3505-pfc-750_8202.dts
@@ -13,7 +13,7 @@
 
 / {
 	model = "WAGO PFC200 750-8202";
-	compatible = "wago,am3505-pfc-750_820x-000c", "wago,am3505-pfc", "ti,am3517", "ti,omap3";
+	compatible = "wago,am3505-pfc-750_820x-000c", "wago,am3505-pfc", "ti,am3517-evm", "ti,am3517", "ti,omap3";
 
 };
 
diff --git a/arch/arm/boot/dts/am3505-pfc-750_8203.dts b/arch/arm/boot/dts/am3505-pfc-750_8203.dts
index 48cf31b..e752669 100644
--- a/arch/arm/boot/dts/am3505-pfc-750_8203.dts
+++ b/arch/arm/boot/dts/am3505-pfc-750_8203.dts
@@ -13,7 +13,7 @@
 
 / {
 	model = "WAGO PFC200 750-8203";
-	compatible = "wago,am3505-pfc-750_820x-000a", "wago,am3505-pfc", "ti,am3517", "ti,omap3";
+	compatible = "wago,am3505-pfc-750_820x-000a", "wago,am3505-pfc", "ti,am3517-evm", "ti,am3517", "ti,omap3";
 
 };
 
diff --git a/arch/arm/boot/dts/am3505-pfc-750_8204.dts b/arch/arm/boot/dts/am3505-pfc-750_8204.dts
index eb54c3c..7f2b441 100644
--- a/arch/arm/boot/dts/am3505-pfc-750_8204.dts
+++ b/arch/arm/boot/dts/am3505-pfc-750_8204.dts
@@ -14,7 +14,7 @@
 
 / {
 	model = "WAGO PFC200 750-8204";
-	compatible = "wago,am3505-pfc-750_820x-000e", "wago,am3505-pfc", "ti,am3517", "ti,omap3";
+	compatible = "wago,am3505-pfc-750_820x-000e", "wago,am3505-pfc", "ti,am3517-evm", "ti,am3517", "ti,omap3";
 
 };
 
diff --git a/arch/arm/boot/dts/am3505-pfc-750_8206.dts b/arch/arm/boot/dts/am3505-pfc-750_8206.dts
index 9778f25..723f379 100644
--- a/arch/arm/boot/dts/am3505-pfc-750_8206.dts
+++ b/arch/arm/boot/dts/am3505-pfc-750_8206.dts
@@ -16,7 +16,7 @@
 
 / {
 	model = "WAGO PFC200 750-8206";
-	compatible = "wago,am3505-pfc-750_820x-000f", "wago,am3505-pfc", "ti,am3517", "ti,omap3";
+	compatible = "wago,am3505-pfc-750_820x-000f", "wago,am3505-pfc", "ti,am3517-evm", "ti,am3517", "ti,omap3";
 
 };
 
diff --git a/arch/arm/boot/dts/am3505-pfc-750_8207.dts b/arch/arm/boot/dts/am3505-pfc-750_8207.dts
index abd6c81..b21b9fa 100644
--- a/arch/arm/boot/dts/am3505-pfc-750_8207.dts
+++ b/arch/arm/boot/dts/am3505-pfc-750_8207.dts
@@ -14,7 +14,7 @@
 
 / {
 	model = "WAGO PFC200 750-8207";
-	compatible = "wago,am3505-pfc-750_820x-001c", "wago,am3505-pfc", "ti,am3517", "ti,omap3";
+	compatible = "wago,am3505-pfc-750_820x-001c", "wago,am3505-pfc", "ti,am3517-evm", "ti,am3517", "ti,omap3";
 
 };
 
-- 
2.7.4

